### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by meteozay**
- Update links to examples
- Update Getting Started and Running Examples
- Fix grammar and typo.

Signed-off-by: Mete Ozay <meteozay@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,269 +99,56 @@ Currently we supports lazy calculation mode to reduce copy of tensors during cal
...
+A transfer learning example with for image classification using the Cifar 10 dataset. TFlite is used for feature extractor and compared with KNN.
 
-The nntrainer is an open source project released under the terms of the Apache License version 2.0.
+#### [Logistic Regression Example](https://github.com/nnstreamer/nntrainer/tree/main/Applications/LogisticRegression)
+
+A logistic regression example using NNTrainer.
+
+## [Getting Started](https://github.com/nnstreamer/nntrainer/blob/main/docs/getting-started.md)
 
-## Contributing
```
Comment (kparichay) [inline README.md:362]: how about not removing this section?

File: README.md [hunk 2]
```diff
@@ -98,269 +99,56 @@ Currently we supports lazy calculation mode to reduce copy of tensors during cal
...
-By default, the command ```ninja -C build`` generates the five example application binaries (Classification, k-NN, LogisticRegression, ReinforcementLearning, and Training) you could try in build/Applications. For 'Training' as an example case,
-```bash
-$ ls build/Applications/Training/jni/ -1
-e189c96@@nntrainer_training@exe
-nntrainer_training
-```
-
-In order to run such example binaries, Tensorflow-lite is a prerequisite. If you are trying to run on the Android, it will automatically download tensorflow (1.9.0) and compile as static library. Otherwise, you need to install it by yourself.
-
-### How to Test
```
Comment (kparichay) [inline README.md:244]: It seems like some of the sections like this are missing from the getting-started.md
As they are being removed from here, can you please add them to getting-started.md as part of this PR

Comment (kparichay) [reply README.md:244]: Or maybe add to how-to-use-testcases.md

File: README.md (hunk 1 above)
Comment (meteozay) [reply README.md:362]: Thank you for the notification, I updated this part recovering the section.

File: README.md (hunk 2 above)
Comment (kparichay) [reply README.md:244]: Thanks for fixing the tests.
`how-to-use-testcases.md` already exists, so you can consider adding it there.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — CHANGES_REQUESTED** [review]
Could you squash the commit into one?

When we make a revision to the patch, we amend the commit and force push

File: README.md [hunk 3]
```diff
@@ -82,10 +83,10 @@ NNTrainer provides
 
 ### Tensor
 
-Tensor is responsible for the calculation of Layer. It executes the addition, division, multiplication, dot production, averaging of Data and so on. In order to accelerate the calculation speed, CBLAS (C-Basic Linear Algebra: CPU) and CUBLAS (CUDA: Basic Linear Algebra) for PC (Especially NVIDIA GPU)  for some of the operation. Later, these calculations will be optimized.
-Currently we supports lazy calculation mode to reduce copy of tensors during calcuation.
+Tensor is responsible for calculation of a layer. It executes several operations such as addition, division, multiplication, dot production, data averaging and so on. In order to accelerate  calculation speed, CBLAS (C-Basic Linear Algebra: CPU) and CUBLAS (CUDA: Basic Linear Algebra) for PC (Especially NVIDIA GPU) are implemented for some of the operations. Later, these calculations will be optimized.
+Currently, we supports lazy calculation mode to reduce complexity for copying tensors during calculations.
```
Comment (wooksong) [inline README.md:87]:
```suggestion
Currently, we support lazy calculation mode to reduce complexity for copying tensors during calculations.
```

File: README.md [hunk 4]
```diff
@@ -82,10 +83,10 @@ NNTrainer provides
 
 ### Tensor
 
-Tensor is responsible for the calculation of Layer. It executes the addition, division, multiplication, dot production, averaging of Data and so on. In order to accelerate the calculation speed, CBLAS (C-Basic Linear Algebra: CPU) and CUBLAS (CUDA: Basic Linear Algebra) for PC (Especially NVIDIA GPU)  for some of the operation. Later, these calculations will be optimized.
-Currently we supports lazy calculation mode to reduce copy of tensors during calcuation.
+Tensor is responsible for calculation of a layer. It executes several operations such as addition, division, multiplication, dot production, data averaging and so on. In order to accelerate  calculation speed, CBLAS (C-Basic Linear Algebra: CPU) and CUBLAS (CUDA: Basic Linear Algebra) for PC (Especially NVIDIA GPU) are implemented for some of the operations. Later, these calculations will be optimized.
```
Comment (wooksong) [inline README.md:86]:
```suggestion
Tensor is responsible for the calculation of a layer. It executes several operations such as addition, division, multiplication, dot production, data averaging, and so on. In order to accelerate calculation speed, CBLAS (C-Basic Linear Algebra: CPU) and CUBLAS (CUDA: Basic Linear Algebra) for PC (Especially NVIDIA GPU) are implemented for some of the operations. Later, these calculations will be optimized.
```

File: README.md [hunk 5]
```diff
@@ -98,262 +99,55 @@ Currently we supports lazy calculation mode to reduce copy of tensors during cal
...
- | Keyward | Loss Name | Description |
+ | Keyword | Loss Name | Description |
  |:-------:|:---:|:---|
  | weight_initializer | Weight Initialization | Xavier(Normal/Uniform), LeCun(Normal/Uniform),  HE(Normal/Unifor) |
  | weight_regularizer | weight decay ( L2Norm only ) | needs set weight_regularizer_constant & type |
  | learnig_rate_decay | learning rate decay | need to set step |
 
 ### APIs
-Currently we privde [C APIs](https://github.com/nnstreamer/nntrainer/blob/master/api/capi/include/nntrainer.h) for Tizen. C++ API also provides soon.
+Currently, we provide [C APIs](https://github.com/nnstreamer/nntrainer/blob/master/api/capi/include/nntrainer.h) for Tizen. C++ API will be also provided soon.
```
Comment (wooksong) [inline README.md:109]:
```suggestion
Currently, we provide [C APIs](https://github.com/nnstreamer/nntrainer/blob/main/api/capi/include/nntrainer.h) for Tizen. C++ API will be also provided soon.
```

File: README.md [hunk 6]
```diff
@@ -5,13 +5,13 @@
...
 
-Even though it trains part of the neural network models, NNtrainer requires quite a lot of functionalities to train from common neural network frameworks. By implementing them, it is good enough to run several examples which can help to understand how it works. There are k-NN, Neural Network, Logistic Regression and Reinforcement Learning with CartPole in Applications directory and some of them use Mobilenet V2 with tensorflow-lite as a feature extractor. All of them tested on Galaxy S8 with Android and PC (Ubuntu 16.04).
+Even though NNTrainer can be used to train sub-models, it requires implementation of additional functionalities to train models obtained from other machine learning and deep learning libraries. In the current version, various machine learning algorithms such as k-Nearest Neighbor (k-NN), Neural Networks, Logistic Regression and Reinforcement Learning algorithms are implemented. We also provide examples for various tasks such as transfer learning of models. In some of these examples, deep learning models such as Mobilenet V2 trained with Tensorflow-lite, are used as feature extractors. All of these were tested on Galaxy S8 with Android and PC (Ubuntu 16.04).
```
Comment (wooksong) [inline README.md:14]:
```suggestion
Even though NNTrainer can be used to train sub-models, it requires implementation of additional functionalities to train models obtained from other machine learning and deep learning libraries. In the current version, various machine learning algorithms such as k-Nearest Neighbor (k-NN), Neural Networks, Logistic Regression, and Reinforcement Learning algorithms are implemented. We also provide examples for various tasks such as transfer learning of models. In some of these examples, deep learning models such as Mobilenet V2 trained with Tensorflow-lite, are used as feature extractors. All of these were tested on Galaxy S8 with Android and PC (Ubuntu 16.04).
```

**Review by zhoonit — APPROVED** [review]
LGTM after resolving all issues above :)
